sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.institutions.controller.getStudentApplicationSummary.e2e-spec.ts

```diff
@@ -0,0 +1,191 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { DataSource, Repository } from "typeorm";
+import {
+  authorizeUserTokenForLocation,
+  BEARER_AUTH_TYPE,
+  createFakeLocation,
+  createTestingAppModule,
+  getAuthRelatedEntities,
+  getInstitutionToken,
+  InstitutionTokenTypes,
+} from "../../../../testHelpers";
+import {
+  createFakeInstitutionLocation,
+  createFakeApplication,
+  saveFakeStudent,
+  createFakeStudentAssessment,
+  createFakeEducationProgramOffering,
+} from "@sims/test-utils";
+import {
+  Application,
+  ApplicationStatus,
+  EducationProgramOffering,
+  Institution,
+  InstitutionLocation,
+  OfferingIntensity,
+  StudentAssessment,
+} from "@sims/sims-db";
+
+describe("StudentInstitutionsController(e2e)-getStudentApplicationSummary", () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  let collegeC: Institution;
+  let collegeCLocation: InstitutionLocation;
+  let applicationRepo: Repository<Application>;
+  let studentAssessmentRepo: Repository<StudentAssessment>;
+  let institutionLocationRepo: Repository<InstitutionLocation>;
+  let educationProgramOfferingRepo: Repository<EducationProgramOffering>;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    appDataSource = dataSource;
+    // College C.
+    const { institution } = await getAuthRelatedEntities(
+      appDataSource,
+      InstitutionTokenTypes.CollegeCUser,
+    );
+    collegeC = institution;
+    collegeCLocation = createFakeInstitutionLocation(collegeC);
```

Comment:
Can we use `College F`(public) instead in preparation for the test changes when the authorization decorator is in place?

Reply:
From the data seed institution type context (Just to make it easy here)

College F - BC Public 
College C - BC Private
College D - BC Private

Reply:
I got it, Actually, I was checking other TCs when I was doing this, and both C and D were used in the public institution tab TCs. So, I randomly chose C 😄 But I got it, I will update it. Thanks